packages/material-ui-lab/src/AvatarGroup/AvatarGroup.d.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { StandardProps } from '@material-ui/core';
+
+export interface AvatarGroupProps
+  extends StandardProps<React.HTMLAttributes<HTMLDivElement>, AvatarGroupClassKey> {
+  /**
+   * The avatars to stack.
+   */
+  children: React.ReactNode;
```

Comment:
A `ReactNode` can be undefined, should probably be `ReactElement`

Reply:
Would it work after `React.Children.toArray(undefined)`?

Reply:
Oh or just:

```tsx
children?: React.ReactNode;
```

Reply:
I had a look at the codebase, we don't have much consistency on this point.